var/spack/repos/builtin/packages/intel-mpi/package.py

```diff
@@ -54,6 +54,13 @@ def mpi_libs(self):
             libraries, root=self.prefix.lib64, shared=True, recurse=True
         )
 
+    @property
+    def mpi_headers(self):
+        # recurse from self.prefix will find too many things for all the
+        # supported sub-architectures like 'mic'
+        return find_headers(
+            'mpi', root='{0}64'.format(self.prefix.include), recurse=False)
```

Comment:
If I do this I will have to check which mpi it is to now which include to use. This way it is transparent for the dependents to mpi. And is is the equivalent to what is done for `mpi_libs`
In fact `<prefix>/include64` in this case is a symbolic link to `<prefix>/linux64/include`